linux_os/guide/system/accounts/accounts-restrictions/root_logins/no_direct_root_logins/rule.yml

```diff
@@ -37,9 +37,10 @@ identifiers:
 references:
     anssi: BP28(R19)
     cis-csc: 1,12,15,16,5
-    cis@alinux3: "5.7"
```

Comment:
I believe these quotes are needed to prevent these references from being converted into `float` numbers.